### PR TITLE
Improvements to checklib

### PR DESCRIPTION
**Pull request by SchrodingersGat**
- Extra checks only generate warnings (not errors)
- Ignores changes in line endings
- Fixed a logic bug where error would be flagged inappropriately

@jkriege2 @pointhi this PR is to improve the Travis output. It should now ignore line-ending changes and also only generate warnings for extra-checking

### CODE REVIEW

**Comment by SchrodingersGat** [comment]
I have also reduced the amount of "noise" the comparelibs script generates

**Comment by SchrodingersGat** [comment]
@jkriege2 can you please quickly check this one so we can fix the Travis issues? :

**Comment by jkriege2** [comment]
Hi!

I can check it this evening ...

Best,
JAN

File: schlib/schlib.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def __init__(self, filename, create = False):
 
         else:
             if not os.path.isfile(self.filename):
-                sys.stderr.write('"{filename}" is not a valid .dcm file\n'.format(filename=self.filename))
+                sys.stderr.write('"{filename}" does not exist'.format(filename=self.filename))
```
Comment (jkriege2) [inline schlib/schlib.py]: please add a linebreak `\n` at the end of this error message, as currently the output is e.g.:

```
"d:\KiCAD\kicad-library\library\Altera.dcm" does not existchecking component: EP1210
  Violating Rule 3.8
    Description and keywords properties contains information about the component.
```

Comment (jkriege2) [inline schlib/schlib.py]: Also: Maybe a better text:
```
  'The DCM-file for library "{filename_lib}" does not exist (expected name: "{filename}").'
```

Basically that's clear, but for newbiew its easier to talk about the .lib, as the editor mostly hides the DCM from them.

**Comment by jkriege2** [comment]
PS: Otherwise it's fine and works for the cases I checked.

best,
JAN

**Comment by SchrodingersGat** [comment]
Should be good to go now @jkriege2

**Comment by jkriege2** [comment]
OK!

nice work  ;-)

JAN

**Pull request merged**